jsonnet/windows_shortcuts.jsonnet

```diff
@@ -214,5 +214,42 @@ local k = import 'lib/karabiner.libsonnet';
            k.input('f4', ['control']),
            k.outputKey('w', ['command']),
            k.condition('if', ['^com\\.google\\.Chrome$'])),
+    // App Shortcuts
+    k.rule('Open Finder with Cmd+1',
```

Comment:
How do you feel about using the pattern "1 (Cmd) [Open Finder]", "2 (Cmd) [Open first pinned Dock app]", "3 (Cmd) [Open second pinned Dock app]", etc. instead of this one?

Reply:
Definitely, it should match the preexisting naming scheme.